From cc068183b2f6dc6032396f4e357f32888c00bd1d Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Sun, 30 Oct 2016 04:00:18 +0100
Subject: [PATCH] lib: Only use local headers in lirc_private.h (#237).

The lirc_private.h header should only be used internally
when building lirc tools, and i this case we should not
care about what's possibly installed in /usr/include/lirc.
---
 lib/config_file.h  |  2 +-
 lib/lirc_private.h | 34 +++++++++++++++++-----------------
 2 files changed, 18 insertions(+), 18 deletions(-)

diff --git a/lib/config_file.h b/lib/config_file.h
index 652bbab..57210f3 100644
--- a/lib/config_file.h
+++ b/lib/config_file.h
@@ -26,7 +26,7 @@
 extern "C" {
 #endif
 
-#include "lirc/ir_remote.h"
+#include "ir_remote.h"
 
 /**
  * Parse a lircd.conf config file.
diff --git a/lib/lirc_private.h b/lib/lirc_private.h
index 741f86c..2336c0a 100644
--- a/lib/lirc_private.h
+++ b/lib/lirc_private.h
@@ -19,22 +19,22 @@
 #include "include/media/lirc.h"
 #endif
 
-#include "lirc/ir_remote_types.h"
-#include "lirc/lirc_log.h"
-#include "lirc/lirc_options.h"
-#include "lirc/lirc-utils.h"
-#include "lirc/curl_poll.h"
-#include "lirc/config_file.h"
-#include "lirc/dump_config.h"
-#include "lirc/input_map.h"
-#include "lirc/driver.h"
-#include "lirc/ir_remote_types.h"
-#include "lirc/drv_admin.h"
-#include "lirc/ir_remote.h"
-#include "lirc/receive.h"
-#include "lirc/release.h"
-#include "lirc/serial.h"
-#include "lirc/transmit.h"
-#include "lirc/ciniparser.h"
+#include "ir_remote_types.h"
+#include "lirc_log.h"
+#include "lirc_options.h"
+#include "lirc-utils.h"
+#include "curl_poll.h"
+#include "config_file.h"
+#include "dump_config.h"
+#include "input_map.h"
+#include "driver.h"
+#include "ir_remote_types.h"
+#include "drv_admin.h"
+#include "ir_remote.h"
+#include "receive.h"
+#include "release.h"
+#include "serial.h"
+#include "transmit.h"
+#include "ciniparser.h"
 
 #endif
-- 
2.5.5

